### PR TITLE
Update smart_proxy_monitoring to 0.3.0

### PR DESCRIPTION
**Pull request by ochnerd**

### CODE REVIEW

**Comment by laugmanuel** [comment]
Duplicate of https://github.com/theforeman/foreman-packaging/pull/10874

File: packages/plugins/rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec [hunk 1]
```diff
@@ -98,26 +69,27 @@ mv %{buildroot}%{gem_instdir}/settings.d/monitoring_icinga2.yml.example \
 mv %{buildroot}%{gem_instdir}/settings.d/monitoring_icingadirector.yml.example \
    %{buildroot}%{foreman_proxy_settingsd_dir}/monitoring_icingadirector.yml
 
-# additional config
-mkdir -p %{buildroot}%{_root_sysconfdir}/foreman-proxy/monitoring
```
Comment (evgeni) [inline packages/plugins/rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec:102]: I think we should keep that (in `_sysconfdir`), as it's the folder the examples place certificates in etc.

Comment (ochnerd) [reply packages/plugins/rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec:102]: So you mean keep it change it from
`%{buildroot}%{_root_sysconfdir}/foreman-proxy/monitoring`
to
`%{buildroot}%{_sysconfdir}/foreman-proxy/monitoring`

Comment (evgeni) [reply packages/plugins/rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec:102]: Yepp! And keep the `%dir` entry in `%files` accordingly.

File: packages/plugins/rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec [hunk 2]
```diff
@@ -98,26 +69,27 @@ mv %{buildroot}%{gem_instdir}/settings.d/monitoring_icinga2.yml.example \
...
 %config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/monitoring_icinga2.yml
 %config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/monitoring_icingadirector.yml
 %license %{gem_instdir}/LICENSE
+%exclude %{gem_instdir}/bundler.d
 %{gem_libdir}
+%exclude %{gem_instdir}/settings.d
 %{foreman_proxy_bundlerd_dir}/%{plugin_name}.rb
 %exclude %{gem_cache}
 %{gem_spec}
-%dir %{_root_sysconfdir}/foreman-proxy/monitoring
```
Comment (evgeni) [inline packages/plugins/rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec:114]: that dir entry ;)

**Review by evgeni — APPROVED** [review]

**Pull request merged**